dz8/controller.py

```diff
@@ -0,0 +1,49 @@
+import operations
+import user_interface as ui
+
+process = True
+ui.start_message()
+i = input()
+while not i == '1' or not i == '2':
+    if i == '1':
+        operations.creating_base()
+        operations.exit_program()
```

Comment:
```suggestion
while True:
    ui.start_message()
    i = input()
    if i == '1':
        operations.creating_base()
```
Сделайте так и уберите функцию `exit_program`. У меня работает как нужно.